script/script.js

```diff
@@ -15,9 +17,42 @@ function AddBookM() {
   domTb.AddRowBooks(s.title, s.author, s.index);
```

Comment:
- [ ] OPTIONAL
The book _title_ and book _author_ input are both accepting numbers as inputs. Kindly endeavor to provide validation for the same